### ISSUE TITLE
Can I pay with European gift cards?

### ISSUE THREAD

**Issue by eggso** [LoukaMB/SynapseX#33]
Hi
So I live in Germany and I'd like to buy Synapse X. Can I still top up with a 20€ gift card or would I not get the product even if the amount is greater when converted? (20€ is around $22) 
If necessary I could even try it with a 25€ gift card

**Comment by Dipek** [comment]
I don't think so, they live in the USA so that might not even be redeemable. + I don't think so that you can redeem European cards in the USA. I paid with PayPal on the sirmeme store (when that was the place to buy S^X with PayPal). And on top of that, I live in Poland and PayPal converts currencies. (Ex, € -> $)
